src/app/pages/admin/admin.component.html

```diff
@@ -37,3 +37,4 @@ <h2 class="heading">Cinemas</h2>
 <app-button (click)="handleAddCinema()">Add cinema +</app-button>
 <h2 class="heading">Extra</h2>
 <app-button>Add extra +</app-button>
+<app-spinner></app-spinner>
```

Comment:
Probably you should add this spinner somewhere in app.component, in order not to duplicate this code